cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -1566,13 +1566,59 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
   });
 
   describe('invalid document structure', () => {
-    describe('normalizable errors', () => {
-      it('does not throw', () => {
-        cy.setInitialValue(invalidDocumentNormalizable);
-        cy.reload();
-        //check that editors content is what we expect (not a thrown error)
-        richText.expectSnapshotValue();
-      });
+    it('accepts document with no content', () => {
+      const emptyDocument = {
```

Comment:
Nit: I would prefer

```suggestion
      const docWithoutContent = {
```

as we explicitly state in [emptyDocument.ts](https://github.com/contentful/rich-text/blob/80315ef5130b79336336ba31de8a55e42bafe319/packages/rich-text-types/src/emptyDocument.ts#L6) what structure is considered to be an empty doc.

So until your structure gets normalized and receives an empty `text` node, it shouldn't really be considered empty.